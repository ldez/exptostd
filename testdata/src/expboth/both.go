package expboth

import (
	"golang.org/x/exp/maps"   // want "Import statement can drop `golang.org/x/exp` prefix"
	"golang.org/x/exp/slices" // want "Import statement can drop `golang.org/x/exp` prefix"
)

func _(m map[string]string, a, b []string) {
	maps.Clone(m)      // want `golang.org/x/exp/maps.Clone\(\) can be replaced by maps.Clone\(\)`
	slices.Equal(a, b) // want `golang.org/x/exp/slices\.Equal\(\) can be replaced by slices\.Equal\(\)`
}
