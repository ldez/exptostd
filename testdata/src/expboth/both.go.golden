package expboth

import (
	"maps"   // want "Import statement 'golang.org/x/exp/maps' can be replaced by 'maps'"
	"slices" // want "Import statement 'golang.org/x/exp/slices' can be replaced by 'slices'"
)

func _(m map[string]string, a, b []string) {
	maps.Clone(m)      // want `golang.org/x/exp/maps.Clone\(\) can be replaced by maps.Clone\(\)`
	slices.Equal(a, b) // want `golang.org/x/exp/slices\.Equal\(\) can be replaced by slices\.Equal\(\)`
}

